test/artifacts/kubecost-checker.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kubecost-checker
+spec:
+  template:
+    spec:
+      volumes:
+      - name: kubecost-test
+        configMap:
+          name: kubecost-test
+      containers:
+      - name: checker
+        image: alpine
+        command: ["/bin/sh"]
+        args: ["/opt/kubecost/kubecost-test.sh"]
+        volumeMounts:
+        - name: kubecost-test
+          mountPath: /opt/kubecost/
+      restartPolicy: Never
+  backoffLimit: 4
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kubecost-test
+data:
+  kubecost-test.sh: |-
+        #!/bin/sh
+        set -euo pipefail
+        apk --no-cache add curl jq
+        curl kommander-kubeaddons-cost-analyzer.kommander.svc:9090
+        curl kommander-kubecost-thanos-query-http.kommander.svc:10902
+
+        echo "Checking aggregatedCostModel api endpoint"
```

Comment:
any way we can log these checks in the teamcity build logs? Right now this tests pretty much only outputs
```
  job.batch/kubecost-checker created
  configmap/kubecost-test created
TestKommanderGroup: kubecost_test.go:36: kubecost checker job succeeded  
```
https://teamcity.mesosphere.io/buildConfiguration/kubeaddons_KommanderAddons/2749790?buildTab=log&focusLine=631&linesState=309

but if you can't, not really a big deal. No bad news is good news, right?

Reply:
unfortunately not, since these checks are part of the yaml file, not the actual e2e test code

Reply:
FWIW the ksphere-testing-framework library offers port-forwarding to services and pods, which can be used to make requests to apps on the cluster directly from the e2e test. KBA tests have some examples, such as the elasticsearch tests: https://github.com/mesosphere/kubernetes-base-addons/blob/master/test/elasticsearch_test.go